frontend/appflowy_flutter/packages/appflowy_editor/lib/src/blocks/base_component/rich_text_with_selection.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+class RichTextWithSelection extends StatefulWidget {
+  const RichTextWithSelection({
+    super.key,
+    required this.text,
+    this.textSelection,
+    this.selectionColor = const Color.fromARGB(100, 33, 149, 243),
```

Comment:
Very arbitrary, maybe we should consider using EditorTheme or something at some point.